docs/ansible/roles/debops.roundcube/upgrade.rst

```diff
@@ -0,0 +1 @@
+.. include:: ../../../../ansible/roles/debops.roundcube/UPGRADE.rst
```

Comment:
The upgrade notes from individual roles will need to be moved to the central location in `docs/news/upgrades.rst` file. In this particular case, I guess the upgrade notes don't need to be included right now, since the role was just introduced in the project. If somebody with an older installation comes over, they will be able to look up the notes in the old location via `git log`.

After the `v0.7.0` release the role directories will be cleaned up from old files (Travis, readmes, etc.).